src/bin/json_output.ts

```diff
@@ -0,0 +1,370 @@
+import {
```

Comment:
Can we move this file outside of `src/bin`? This directory is expected to have executable files only (with shebang prefix). Logic in this file, however is mostly utility.

Reply:
Moved to src/util. Fixed in 6d2b8b6.